{"message":"Something went wrong, try again!","success":false,"response":{"traceback":"Traceback (most recent call last):\n  File \"/home/GetLinkBackend/helpers/utils/advice.py\", line 25, in wrapper\n    return func(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/GetLinkBackend/talent_dashboard/views.py\", line 3524, in post\n    cover_letter_subject, cover_letter = generate_cover_letter(data, resume)\n                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/GetLinkBackend/helpers/reusable.py\", line 858, in generate_cover_letter\n    response_data = json.loads(chatgpt_response)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/json/__init__.py\", line 346, in loads\n    return _default_decoder.decode(s)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib/python3.12/json/decoder.py\", line 353, in raw_decode\n    obj, end = self.scan_once(s, idx)\n               ^^^^^^^^^^^^^^^^^^^^^^\njson.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 2 column 5 (char 6)\n"}}